examples/wkwebview/wkwebview/LinkViewController.swift

```diff
@@ -69,62 +71,81 @@ class LinkViewController: UIViewController, WKNavigationDelegate {
     }
     
     func webView(_ webView: WKWebView,
-                 decidePolicyForNavigationAction navigationAction: WKNavigationAction,
-                 decisionHandler: ((WKNavigationActionPolicy) -> Void)) {
+                 decidePolicyFor navigationAction: WKNavigationAction,
+                 decisionHandler: @escaping ((WKNavigationActionPolicy) -> Void)) {
```

Comment:
sure -- I guess the point is to clearly indicate functionality above all else. makes sense.